src/data/actions/assets.js

```diff
@@ -70,10 +75,15 @@ export const deleteAsset = (assetId, courseDetails) =>
     clientApi.requestDeleteAsset(courseDetails.id, assetId)
       .then((response) => {
         if (response.ok) {
-          dispatch(deleteAssetSuccess(assetId, response));
-        } else {
-          dispatch(assetDeleteFailure(response));
+          return response.json();
```

Comment:
I have changed the response to be JSON.

Reply:
Question here -- if the response from the API to delete the asset comes back with a non 2XX response, does it return any additional info as to why it failed to delete in the body? i.e could we still use a `response.json()` to note a more verbose error message to the user in the future? Or do all non-2XX responses return an empty body?

Reply:
The API only returns 204 or 404, neither of which have bodies, so I have actually removed the call to jsonify the response. Does that seem like the right call?